codalab/worker/docker_image_manager.py

```diff
@@ -38,44 +36,16 @@ def __init__(self, commit_file, max_image_cache_size, max_image_size):
         :param max_image_cache_size: Total size in bytes that the image cache can use
         :param max_image_size: Total size in bytes that the image can have
         """
+        super().__init__(max_image_size, max_image_cache_size)
```

Comment:
Update https://github.com/codalab/codalab-worksheets/blob/ea6f19297309eeebac508f00855f08637ea75ad9/codalab/worker/main.py#L93 to 1) make descriptions of args more generic and 2) specify that max image size does not work with singularity.